slitlessutils/core/wfss/data/wfsscollection.py

```diff
@@ -398,24 +432,32 @@ def from_list(cls, filenames, filename='<LIST>'):
 
         """
 
-        LOGGER.info('Loading WFSS data from python list')
+        LOGGER.info(f'Loading WFSS data from file list: {obsfile}')
+
+        filenames = []
+        with open(obsfile, 'r') as fp:
+            for line in fp:
+                line = line.strip()
+                if line and line[0] not in cls.COMMENTS:
+                    filenames.append(line)
+
+        obj = cls.from_list(filenames, filename=obsfile)
 
-        obj = cls(filetype='observed', filename=filename)
-        for f in filenames:
-            obj[f] = ObservedData(f)
         return obj
 
     @classmethod
-    def from_file(cls, obsfile):
+    def from_list(cls, filenames, filename='<list>'):
```

Comment:
```suggestion
    def from_list(cls, filenames, filename='<LIST>'):
```
Nitpicky, but might as well match the capitalization from the docstring below.